### ISSUE TITLE
Documentation build is failing

### ISSUE THREAD

**Issue by cphalpert** [spotify/chartify#42]
https://readthedocs.org/projects/chartify/builds/

**Comment by g-walsh** [comment]
It seems this is caused by a build fail due to incompatible dependencies. When running `make install`

>jupyter-console 6.0.0 has requirement prompt-toolkit<2.1.0,>=2.0.0, but you'll have prompt-toolkit 1.0.15 which is incompatible.

When installing `jupyter`, `jupyter-console` 6.0.0 requires `prompt-toolkit` 2.0/2.1

When `b0b2e12` was merged there was an incompatibility between ipython and jupyter-console so a requirement on `ipython` 6.5.0 and `prompt-toolkit` 1.0.15 was made.

The build runs fine for me locally if you either rollback the requirement of `jupyter-console` to 5.2.0 meaning:

- jupyter-console==5.2.0
- ipython==6.5.0
- prompt-toolkit==1.0.15

or since the problem has since been patched, upgrade everything and use `prompt-toolkit` 2

- jupyter-console>=6.0.0
- ipython>=7.0.0
- prompt-toolkit>=2.0.0

I assume the documentation builds will work after that as that seems to be where the log stops.

**Comment by soumikbaithalu** [comment]
Is the issue still open ??

**Comment by valleyofblackpanther** [comment]
yeah seems like it, i don't think there's anything to do on this issue.
